packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -374,10 +374,14 @@ abstract class ScrollView extends StatelessWidget {
   final DragStartBehavior dragStartBehavior;
 
   /// {@template flutter.widgets.scroll_view.keyboardDismissBehavior}
-  /// [ScrollViewKeyboardDismissBehavior] the defines how this [ScrollView] will
+  /// [ScrollViewKeyboardDismissBehavior] defines how this [ScrollView] will
   /// dismiss the keyboard automatically.
   /// {@endtemplate}
-  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  ///
+  /// If [keyboardDismissBehavior] is null then it will fallback to
+  /// [scrollBehavior] or to the closest [ScrollConfiguration] provided
+  /// by the context.
```

Comment:
```suggestion
  /// If [keyboardDismissBehavior] is null then it will fallback to
  /// [scrollBehavior]. If that is also null, the inherited 
  /// [ScrollBehavior.keyboardDismissBehavior] will be used.
```